routes/data.js

```diff
@@ -248,3 +249,38 @@ exports.organizationsPost = async function (req, res) {
     .update(newOrgData);
   res.redirect('/dashboard');
 };
+
+exports.updatePlaceIDs = async function (req, res) {
+  // Get all entries from the Organizations table.
+  const organizations = await firestore.collection(resolveCollectionName('Organizations')).get();
+
+  let tasksCompleted = '';
+
+  await Promise.all(
+    organizations.docs.map(async (doc) => {
+      const oldPlaceID = doc.data().placeID;
+      const oldPlaceName = doc.data().name;
+
+      if (oldPlaceID != undefined && oldPlaceID != '') {
+        // If the current entry has a placeID, check if it is the latest one.
+        const placeJSON = await (
```

Comment:
Why do you need two nested ```await```s?

Reply:
Both the fetch() and res.json() functions need awaits: https://gist.github.com/msmfsd/fca50ab095b795eb39739e8c4357a808#gistcomment-2085059